src/main/java/mysutong/Storage.java

```diff
@@ -0,0 +1,66 @@
+package mysutong;
+
+import mysutong.Deadline;
+import mysutong.Event;
+
+import java.io.*;
```

Comment:
Imports should be made explicit according to the coding standard.
```suggestion
import java.io.File;
import java.io.FileNotFoundException;
import java.io.BufferedWriter;
import java.io.IOException;
```